src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -1001,6 +1016,8 @@ internal sealed partial class BoundDeconstructionAssignmentOperator : BoundExpre
         // TODO: implement IOperation for pattern-matching constructs (https://github.com/dotnet/roslyn/issues/8699)
         protected override OperationKind ExpressionKind => OperationKind.None;
 
+        protected override ImmutableArray<IOperation> Children => ImmutableArray.Create<IOperation>(this.Left, this.Right);
```

Comment:
>this.Right [](start = 111, length = 10)

We might not want to expose the right node as is. For example, I believe it can contain special conversions that probably shouldn't be exposed to the outside world in that form. I think we might want to expose only the nodes that aren't special for deconstruction. CC @jcouv